package.json

```diff
@@ -80,7 +80,6 @@
 	},
 	"ava": {
 		"workerThreads": false,
-		"concurrency": 1,
```

Comment:
It appears this PR removing the race condition made this workaround unnecessary.